docs/io/ascii/howtoexamples.rst

```diff
@@ -0,0 +1,209 @@
+.. include:: references.txt
+
+.. _io_ascii_how_to_examples:
+
+How to fix a table that is not read or written correctly
```

Comment:
That's because I never finished what I indented to write here. This is supposed to get to e.g. https://github.com/astropy/astropy/pull/3812#issuecomment-314771680 which I think *is* useful - there is no guarantee that you and I will be on this project forever, so we should write it down. It can also be a path for others to become developers - that's how I got into astropy (with your help - remember the DAOPhotTableReader?)